mip/example_constraint_test.go

```diff
@@ -38,22 +38,24 @@ func ExampleConstraint_equal() {
 func ExampleConstraint_lessThanOrEqual() {
 	model := mip.NewModel()
 
-	c, _ := model.NewConstraint(mip.LessThanOrEqual, 1.0)
-
+	c := model.NewConstraint(mip.LessThanOrEqual, 1.0)
+	c.SetName("constraint")
```

Comment:
Just to be clear, calling `SetName` isn't _required_, right? I'm not sure if these examples show up in the docs or not. If they do, I think we should only call `SetName` in a single example showing how to use the string representation of a model.

Reply:
It is not required, it reverts to what it already was (and now I realize a Float should revert to F{index} and not C{index})